### DISCUSSION TITLE
pole figure size error due to colorbar

### DISCUSSION THREAD

**Discussion by amirtopaz** [mtex-toolbox/mtex#961] [Ask Anything] [answered]
Hello Dear Mtex Colleagues

I plot a pole figure using the command plotPDF, consisted of three Miller indices 100,010 and 001.
when I add a color bar to the plot, the last pole figure (001), gets smaller because the color bar is attached to it and impacts its size (see attached images).
Does anybody know how to fix this?

The code I use is:
figure 
plotPDF(odf_hbl,h_hbl,'colorrange',[0 7],'antipodal');
colorbar
* if I write mtexColorbar instead of colorbar it doesn't have any effect

![cpo without colorbar](https://user-images.githubusercontent.com/62837798/114677237-c846fe00-9d12-11eb-87a9-a0a89061aa3b.jpg)
![cpo with colorbar](https://user-images.githubusercontent.com/62837798/114677251-c9782b00-9d12-11eb-980b-c82933330248.jpg)

**Comment by ralfHielscher** [comment]
Have a look here: https://mtex-toolbox.github.io/ColorMaps.html

Does this work for you?

Ralf.

**Comment by kilir** [comment, marked as answer]
As far as I remember, this was a bug maybe a year ago. Or are you experiencing this on the current version of Mtex?
Rüdiger

**Reply by amirtopaz** [reply]
ok, it works now.. I changed the command colorbar to mtexColorbar once again and now it works fine...
I guess the latest mtex version fixed this bug when using the command mtexColorbar, and I haven't tried it for the latest version until now.
Thanks !
